### PR TITLE
[12.0][ADD] - Add analytic brand module

### PR DESCRIPTION
**Pull request by sbejaoui**
associate an analytic account to a brand that will be used as a default value where the brand is used if the analytic accounting is activated.

### CODE REVIEW

File: account_brand/models/account_invoice.py [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-from odoo import models
+from odoo import api, models
 
 
 class AccountInvoice(models.Model):
     _name = 'account.invoice'
     _inherit = ['account.invoice', 'res.brand.mixin']
+
+    @api.onchange('brand_id', 'invoice_line_ids')
+    def _onchange_brand_id(self):
```
Comment (dreispt) [inline account_brand/models/account_invoice.py:21]: Are you sure this is triggered in the case the invoice is created from a Sale Order, and the brand is carried over from there?

Comment (sbejaoui) [reply account_brand/models/account_invoice.py:21]: This onchange must not be triggered at creation as the analytic account is set in the sale order an carried with the standard code.

We keep the possibility for the user to unset the analytic account in the sale order and have a branded invoice without any analytic.

**Review by qgroulard — APPROVED** [review]
LGTM (code review)

**Review by gva-acsone — APPROVED** [review]
LGTM functional test

**Pull request merged**